test/src/commands.spec.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+const fs = require('fs')
+const commands = require('src/commands')
+const sinon = require('sinon')
+
+describe('commands', () => {
+  describe('tasks', () => {
+    const confPath = `${process.cwd()}/devlab.yml`
+
+    before(() => {
+      sinon.spy(console, 'log')
+      fs.writeFileAsync(confPath, [
+        'tasks:',
+        '  stuff: echo "stuff" > /dev/null',
+        '  lint: eslint .',
+        '  ci: stuff test'
+      ].join('\n'))
+    })
+
+    after(() => {
+      console.log.restore()
+      fs.unlinkAsync(confPath)
+    })
+
+    it('pretty prints all tasks in the config', () => {
+      commands.tasks()
+      expect(console.log).to.have.been.calledWithExactly([
```

Comment:
Would be nicer here to spy on the `output` module's `log` method, if one existed.  I decided not to implement it in this PR as there are several other places using the console log.  If we decide to go with one, we can do a proper refactor in another PR.